dids/src/main/kotlin/web5/sdk/dids/Models.kt

```diff
@@ -0,0 +1,14 @@
+package web5.sdk.dids
+
+import com.fasterxml.jackson.annotation.JsonValue
+
+/**
+ * Enum representing the purpose of a public key.
+ */
+public enum class PublicKeyPurpose(@get:JsonValue public val code: String) {
```

Comment:
@andresuribe87 we should consider killing this since it's present in the DID lib we use